### PR TITLE
Awesome books: adding ES6

### PR DESCRIPTION
**Pull request by jevazquezb**
## Awesome books: Restructuring the app code

In this project we:
- Used ES6 modules to write modular JavaScript.
- Learned how to use proper ES6 syntax.
- Understood improvements that ES6 brings to JavaScript.
- Used npm as software packages system.

### CODE REVIEW

File: styles/index.css [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  padding: 0.5rem 0.5rem 0.5rem 1.7rem;
+}
+
+.logo {
+  font-size: 1.3rem;
+  font-weight: bold;
+}
+
+.menu {
+  display: flex;
```
Comment (ahmetbozaci) [inline styles/index.css]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
This is just an example if you have some CSS rules please combine them.
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: styles/index.css [hunk 2]
```diff
@@ -0,0 +1,177 @@
+body {
+  display: flex;
+  flex-direction: column;
```
Comment (ahmetbozaci) [inline styles/index.css:10]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
This is just an example if you have same CSS rules please combine them.
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: styles/index.css [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+}
+
+.rmbtn:hover {
+  cursor: pointer;
+}
+
+.form {
+  display: none;
+  flex-direction: column;
+  align-items: center;
```
Comment (ahmetbozaci) [inline styles/index.css]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
This is just an example if you have same CSS rules please combine them.
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @jevazquezb  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- Everything is working as expected.
- You know very well how to use ES6.
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a 

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**